packages/surveys/src/styles/global.css

```
@@ -62,33 +62,33 @@ p.fb-editor-paragraph {
+  --subheading-color: var(--slate-700);
+  --placeholder-color: var(--slate-300);
+  --info-text-color: var(--slate-500);
+  --signature-text-color: var(--slate-500);
+  --branding-text-color: var(--slate-500);
+  --survey-background-color: white;
+  --survey-border-color: var(--slate-50);
+  --accent-background-color: var(--slate-200);
+  --accent-background-color-selected: var(--slate-100);
+  --input-background-color: var(--slate-50);
+  --input-background-color-selected: var(--slate-200);
+  --placeholder-color: var(--slate-400);
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--placeholder-color is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)